mvcc/backend/backend.go

```diff
@@ -163,6 +165,20 @@ func (b *backend) ForceCommit() {
 }
 
 func (b *backend) Snapshot() Snapshot {
+	stopc, donec := make(chan struct{}), make(chan struct{})
+	go func() {
+		defer close(donec)
+		start := time.Now()
+		for {
+			select {
+			case <-time.After(snapshotWarningTimeout):
+				plog.Warningf("snapshotting is taking more than %v seconds to finish [started at %v]", time.Since(start).Seconds(), start)
```

Comment:
I'll update this with  ` time.NewTicker`. 

I am going to have another pr for track snapshotting duration as a Prometheus metric, https://github.com/coreos/etcd/issues/7878.

So probably no need to printing total duration.